thirdeye/thirdeye-pinot/src/main/resources/assets/css/thirdeye.css

```diff
@@ -427,21 +455,40 @@ text {
   line-height: 32px;
   border-radius: 2px;
   transition-duration: 167ms;
-  transition-property: background-color,color;
+  transition-property: background-color, box-shadow, color;
   transition-timing-function: cubic-bezier(0,0,.2,1);
 }
 
 .thirdeye-btn:hover,
 .thirdeye-btn:focus {
   background-color: #0073b1;
+  font-weight: 500;
   color: white;
 }
 
-.thirdeye-btn:active{
+.thirdeye-btn:active {
   color: white;
   background-color: #006097;
 }
 
+.thirdeye-btn--secondary {
+  background-color: transparent;
```

Comment:
I'm not sure if we should be copying this from art deco. It might make sense just to use bootstrap buttons and restyle them without transitions?